bindings/stronghold-nodejs/src/account/storage/stronghold.rs

```diff
@@ -72,28 +70,31 @@ impl NapiStronghold {
 
   /// Creates a new keypair at the specified `location`
   #[napi]
-  pub async fn key_new(&self, did: &NapiDID, location: &NapiKeyLocation) -> Result<String> {
+  pub async fn key_new(&self, did: &NapiDID, location: &NapiKeyLocation) -> Result<Vec<u32>> {
     let public_key: PublicKey = self.0.key_new(&did.0, &location.0).await.napi_result()?;
-    Ok(encode_b58(&public_key))
+    let public_key: Vec<u8> = (&public_key).into();
+    Ok(public_key.into_iter().map(u32::from).collect())
   }
 
   /// Inserts a private key at the specified `location`.
   #[napi]
-  pub async fn key_insert(&self, did: &NapiDID, location: &NapiKeyLocation, private_key: String) -> Result<String> {
-    let private_key: PrivateKey = decode_b58(&private_key).napi_result()?.into();
+  pub async fn key_insert(&self, did: &NapiDID, location: &NapiKeyLocation, private_key: Vec<u32>) -> Result<Vec<u32>> {
```

Comment:
Can we open an issue with napi.rs regarding `Vec<u8>`/`UInt8Array` support? I don't understand the problem in enough detail to do so myself.

Reply:
Totally not required, as long as we have a way around this.

Reply:
[issue](https://github.com/napi-rs/napi-rs/issues/1105).